nixos/modules/services/misc/gitea.nix

```diff
@@ -200,6 +206,18 @@ in
         description = "HTTP listen port.";
       };
 
+      enableSocket = mkOption {
```

Comment:
What does everyone think about `enableUnixSocket` instead, like the `memcached` module?

Reply:
Thanks, renamed.